pkgs/development/libraries/libdatachannel/default.nix

```
@@ -0,0 +1,61 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, ninja
+, pkg-config
+, libnice
+, nlohmann_json
+, openssl
+, srtp
+, usrsctp
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libdatachannel";
+  version = "0.18.4";
+
+  src = fetchFromGitHub {
+    owner = "paullouisageneau";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-kSK+5gFMG6tq89R1m08gNBKPdwyR/mLEDhWXQ/uk34o=";
+    fetchSubmodules = true;
+  };
+
+  outputs = [ "out" "dev" ];
+
+  strictDeps = true;
+  nativeBuildInputs = [
+    cmake
+    ninja
+    pkg-config
+  ];
+  buildInputs = [
+    libnice
+    nlohmann_json
+    openssl
+    srtp
+    usrsctp
+  ];
+
+  cmakeFlags = [
+    "-DUSE_NICE=ON"
+    "-DUSE_SYSTEM_JSON=ON"
+    "-DUSE_SYSTEM_SRTP=ON"
+    "-DUSE_SYSTEM_USRSCTP=ON"
+  ];
+
+  # FIXME: upstream cmake files shouldn't be relative
+  postFixup = ''
+    cp -r $out/lib/libdatachannel.so.0.18.4 $dev/lib/
+  '';
```

```suggestion
  postFixup = ''
    # Fix shared library path that will be incorrect on move to "dev" output
    substituteInPlace "$dev/lib/cmake/LibDataChannel/LibDataChannelTargets-release.cmake" \
      --replace "\''${_IMPORT_PREFIX}/lib" "$out/lib"
  '';
```
This pattern turns up a lot when running `git grep _IMPORT_PREFIX`.

---

Thanks! This was the fix I was searching for.